extensions/tmtv/src/utils/measurementServiceMappings/utils/getSOPInstanceAttributes.js

```diff
@@ -0,0 +1,69 @@
+import * as cornerstone from '@cornerstonejs/core';
```

Comment:
Shouldn't we just expose the `metaData` object through the cornrerstone extensions utilities? Its going to be used pretty commonly I expect?